docs/manifest.json

```diff
@@ -246,575 +565,791 @@
+                                "href": "/docs/elements/guides/sign-in"
+                              },
+                              {
+                                "title": "Build a sign-up flow",
+                                "href": "/docs/elements/guides/sign-up"
+                              },
+                              {
+                                "title": "Styling",
+                                "href": "/docs/elements/guides/styling"
+                              }
+                            ]
+                          ]
                         },
                         {
-                          "title": "Ruby / Rails",
-                          "href": "/docs/backend-requests/handling/ruby-rails",
-                          "icon": "ruby"
+                          "title": "Examples",
+                          "items": [
+                            [
+                              {
+                                "title": "Sign-in",
+                                "href": "/docs/elements/examples/sign-in"
+                              },
+                              {
+                                "title": "Sign-up",
+                                "href": "/docs/elements/examples/sign-up"
+                              },
+                              {
+                                "title": "Primitives",
+                                "href": "/docs/elements/examples/primitives"
+                              },
+                              {
+                                "title": "shadcn/ui",
+                                "href": "/docs/elements/examples/shadcn-ui"
+                              }
+                            ]
+                          ]
                         },
                         {
-                          "title": "Manual JWT Verification",
-                          "href": "/docs/backend-requests/handling/manual-jwt"
+                          "title": "Component Reference",
+                          "items": [
+                            [
+                              {
+                                "title": "Common",
+                                "href": "/docs/elements/reference/common"
+                              },
+                              {
+                                "title": "Sign-in",
+                                "href": "/docs/elements/reference/sign-in"
+                              },
+                              {
+                                "title": "Sign-up",
+                                "href": "/docs/elements/reference/sign-up"
+                              }
+                            ]
+                          ]
                         }
                       ]
                     ]
                   },
                   {
-                    "title": "Versioning",
+                    "title": "⚠ Custom Flows",
```

Comment:
The ⚠ emoji/unicode will render differently (or not at all) and won't fit in with the rest of the icons. So I think you need to use the `icon` property and get that icon created (if not existent already).

Why do you think we need to add this icon there?

Reply:
I don't think there is a ton of value with the icon -- I'm not sure what its trying to say to the reader.

Reply:
👍  makes sense. in some of our future designs its a bit more relevant, because we show prominent warnings for all custom flows - telling the user that this is the advanced way to do things and not the best/easiset... but that work isn't part of this speedrun

Reply:
We could also move it under "advanced"...

Reply:
yeah, i like that idea

Reply:
Ok, will shift that and get this merged! 🎉